### PR TITLE
style: add update method for various resources in emqx statefulSet

### PR DESCRIPTION
**Pull request by Gala-R**
the resources involved are acl, pluginsConfig, loadedPlugins, license .

### CODE REVIEW

**Comment by Rory-Z** [comment]
not ack, is acl

**Comment by Rory-Z** [comment]
One unit test function should only test one function, not both `generate` function and `update` function

File: controllers/apps/v1beta3/emqx_handler.go [hunk 1]
```diff
@@ -678,43 +615,20 @@ func generateLoadedModules(instance appsv1beta3.Emqx, sts *appsv1.StatefulSet) (
...
 	names := appsv1beta3.Names{Object: instance}
 	emqxEnterprise, ok := instance.(*appsv1beta3.EmqxEnterprise)
 	if !ok {
-		return nil, sts
+		return nil
 	}
-	if reflect.ValueOf(emqxEnterprise.GetLicense()).IsZero() {
-		return nil, sts
+	license := emqxEnterprise.GetLicense()
+	if isEmpty(license) {
```
[the remark this answers is not in the document]
Comment (Gala-R) [reply controllers/apps/v1beta3/emqx_handler.go:628]: what will be nil pointer? the type of  license is struct.

[the remark this answers is not in the document]
Comment (Rory-Z) [reply controllers/apps/v1beta3/emqx_handler.go:628]: Why this is failed: https://github.com/emqx/emqx-operator/runs/7762749853?check_suite_focus=true

[the remark this answers is not in the document]
Comment (Gala-R) [reply controllers/apps/v1beta3/emqx_handler.go:628]: the return of generateAcl maybe nil pointer,

**Review by Rory-Z — APPROVED** [review]

**Pull request merged**